src/plot_api/plot_api.js

```diff
@@ -3706,17 +3706,29 @@ function purge(gd) {
     return gd;
 }
 
+// determines if the graph div requires a recalculation of its inverse matrix transforms by comparing old + new bounding boxes.
+function calcInverseTransform(gd, newBBox) {
```

Comment:
This function never called using the second argument. So I think you could/should drop it.

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/5302/commits/0b2397ad8627b0f19df3596deebac22de6f798fa.